### PR TITLE
Adds wemake-python-styleguide

### PR DESCRIPTION
**Pull request by sobolevn**

### CODE REVIEW

**Comment by hugovk** [comment]
Thanks for the PR!

Is `wemake-python-styleguide` a style guide? I know it has `styleguide` in the name, but https://github.com/wemake-services/wemake-python-styleguide says it's a linter.

**Comment by sobolevn** [comment]
@hugovk it is both a styleguide and a linter. We declare how to write `python` code in out company and also check that people obey our rules with linter.

**Comment by hugovk** [comment]
Please could you:

* > Add a comment after the link to state why you think it is awesome, and briefly describing it.

https://github.com/RichardLitt/awesome-styleguides/blob/master/contributing.md

File: readme.md [hunk 1]
```diff
@@ -169,6 +169,7 @@ If you have used any of these styleguides, please add a comment if you can! The
 * [PEP 8 Offical Style Guide for Python](https://pep8.org/)
 * [PEP 257 - Docstring Conventions](http://legacy.python.org/dev/peps/pep-0257/)
 * [The Hitchhiker's Guide to Python](http://docs.python-guide.org/en/latest/writing/style/)
+* [wemake-python-styleguide](https://github.com/wemake-services/wemake-python-styleguide) - The strictest and most opinionated python styleguide and linter ever.
```
Comment (hugovk) [inline readme.md]:
```suggestion
* [wemake-python-styleguide](https://github.com/wemake-services/wemake-python-styleguide) - The strictest and most opinionated Python styleguide and linter ever.
```

**Comment by sobolevn** [comment]
@hugovk that's actually the first time I have used suggested edits! 🚀

**Comment by hugovk** [comment]
Yes, suggested edits are really good! Thanks for the PR!

**Pull request merged**